src/NzbDrone.Core/MediaFiles/MovieImport/ImportApprovedMovie.cs

```diff
@@ -116,11 +116,10 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
                             break;
                     }
 
+                    movieFile.OriginalFilePath = GetOriginalFilePath(downloadClientItem, localMovie);
+                    movieFile.SceneName = GetSceneName(downloadClientItem, localMovie);
                     if (newDownload)
                     {
-                        movieFile.OriginalFilePath = GetOriginalFilePath(downloadClientItem, localMovie);
-                        movieFile.SceneName = GetSceneName(downloadClientItem, localMovie);
```

Comment:
I use OriginalTitle for Movie file renaming, and if scenename is not set it's not working correctly and you have to manually edit the database to update the scenename.

This happens for example when you switch to Radarr and import your existing movies or have to manually import a movie because you downloaded it outside of Radarr.

I think newDownload is true when you manually import a movie in queue if it failed to match a movie, so it's working fine there.

What's the problem setting the Scenename from the filename when it has been manually downloaded and imported? It's still a valid scenename, it just wasn't a fresh download.